docs/examples/en/controls/OrbitControls.html

```diff
@@ -282,6 +282,11 @@ <h3>[method:radians getPolarAngle] ()</h3>
 			Get the current vertical rotation, in radians.
 		</p>
 
+		<h3>[method:radians getDistance] ()</h3>
+		<p>
+			Get the current dolly distance ( [page:PerspectiveCamera] only ).
```

Comment:
Returns the distance from the camera to the target.

@Mugen87 Do you agree with this suggestion?

Reply:
👍